src/EmailObject.js

```diff
@@ -10,7 +10,9 @@ module.exports = class EmailObject {
       to: emailParams.recipients,
       cc: emailParams.cc,
       bcc: emailParams.bcc,
-      reply_to: emailParams.replyTo,
+      reply_to: {
+        email: emailParams.replyTo
+      },
```

Comment:
This would not allow anyone set a `reply_to.name` as per docs https://developers.mailersend.com/api/v1/email.html#send-an-email